### PR TITLE
pythonPackages.sphinxcontrib-applehelp: init at 1.0.1

### PR DESCRIPTION
**Pull request by wamserma**
###### Motivation for this change

build dependency of sphinx 2.2.0

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [X] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/development/python-modules/sphinxcontrib-applehelp/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+buildPythonPackage rec {
+  pname = "sphinxcontrib-applehelp";
+  version = "1.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "edaa0ab2b2bc74403149cb0209d6775c96de797dfd5b5e2a71981309efab3897";
+  };
+
+  propagatedBuildInputs = [  ];
```
Comment (jonringer) [inline pkgs/development/python-modules/sphinxcontrib-applehelp/default.nix]: just remove, if not needed
```suggestion
```

File: pkgs/development/python-modules/sphinxcontrib-applehelp/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, sphinx
```
Comment (jonringer) [inline pkgs/development/python-modules/sphinxcontrib-applehelp/default.nix]: not used
```suggestion
```

**Review by jonringer — COMMENTED** [review]
minor suggestions, otherwise LGTM

**Comment by wamserma** [comment]
superseded by https://github.com/NixOS/nixpkgs/pull/71601

**Pull request closed without merging**